code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -1039,8 +1063,11 @@
 
 ///Proc to turn bodypart into another.
 /obj/item/bodypart/proc/change_bodypart(obj/item/bodypart/new_type)
+	RETURN_TYPE(/obj/item/bodypart)
+
 	var/mob/living/carbon/our_owner = owner //dropping nulls the limb
-	drop_limb(TRUE)
 	var/obj/item/bodypart/new_part = new new_type()
-	new_part.attach_limb(our_owner, TRUE)
-	qdel(src)
+	new_part.replace_limb(our_owner, TRUE)
+
+/obj/item/bodypart/proc/access_var(var_name)
+	return vars[var_name]
```

Comment:
What's this used for?

Reply:
Accidentally left over from screwing around with protected vars, oops!